src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.AspNetCore/TargetingHttpContextMiddleware.cs

```diff
@@ -50,7 +47,12 @@ public async Task InvokeAsync(HttpContext context, ITargetingContextAccessor tar
 
             if (targetingContext != null)
             {
-                context.Items[TargetingIdKey] = targetingContext.UserId;
+                var requestTelemetry = context.Features.Get<RequestTelemetry>();
```

Comment:
NIT: can we rename `context` to `httpContext`? There are so many "contexts". Just make it a little bit easier to read the code :)

Reply:
Good call, yes.